tools/read_mutation.hh

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2021-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include <filesystem>
+#include <functional>
+#include <string_view>
+#include <optional>
+
+#include "tools/sstable_manager_service.hh"
```

Comment:
I think instead of a separate header file, the declaration of `sstable_manager_service` could be just added below. It is very tightly tied to `read_mutation_from_table_offline()`, it doesn't have any generic uses, beyond just creating one for this method.

Reply:
sure.